examples/udp-client/udp_client.description

```diff
@@ -0,0 +1,5 @@
+// The UDP Client is used to send data to a specified remote host.
```

Comment:
"to a specific remote host using the UDP protocol".

Reply:
updated